tests/modules/restful_test/restful_test.install

```diff
@@ -54,5 +54,37 @@ function restful_test_schema() {
     'primary key' => array('pid'),
   );
 
+  // Defining table via hook_install() due to drupal_write_record().
```

Comment:
Does this mean that if we are not declaring the table via `hook_schema` we cannot use `drupal_write_record`?